addons/mail/static/src/components/composer/composer.xml

```diff
@@ -160,7 +160,7 @@
                         'o-composer-is-compact': props.isCompact,
                         'o-has-current-partner-avatar': props.hasCurrentPartnerAvatar,
                     }"
-                    t-att-disabled="!composer.canPostMessage ? 'disabled' : ''"
+                    t-att-disabled="(!composer.canPostMessage || composer.isPostingMessage) ? 'disabled' : ''"
```

Comment:
I think we should update `canPostMessage` to false if `isPostingMessage` is true. Then we don't even need a new condition on the component js file, and no diff here either.